about.html

```diff
@@ -0,0 +1,122 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="stylesheet" href="css/about.css">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/css/bootstrap.min.css"
+    integrity="sha384-Gn5384xqQ1aoWXA+058RXPxPg6fy4IWvTNh0E263XmFcJlSAwiGgFAW/dAiS6JXm" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
+  <link rel="preconnect" href="https://fonts.googleapis.com" />
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@300;400;700&display=swap" rel="stylesheet" />
+  <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet" />
+  <!-- fontawesome -->
+  <script src="https://kit.fontawesome.com/d31edfa6bf.js" crossorigin="anonymous"></script>
+  <title>Capstone Project</title>
+</head>
+
+<body>
+  <header class='header'>
+    <div class='header-links'>
+      <ul class='bar-menu'>
+        <li class="bar-item"><a href="#" class='nav-link'><i class="fa fa-facebook" aria-hidden="true"></i></a></li>
+        <li class="bar-item"><a href="#" class='nav-link'><i class="fa fa-twitter" aria-hidden="true"></i></a></li>
+        <li class="bar-item"><a href="#" class='nav-link'>English</a></li>
+        <li class="bar-item"><a href="#" class='nav-link'>My Page</a></li>
+        <li class="bar-item"><a href="#" class='nav-link'>Logout</a></li>
+      </ul>
+      <div class="hamburger1">
+        <span class="bar1"></span>
+        <span class="bar1"></span>
+        <span class="bar1"></span>
+      </div>
+    </div>
+    <nav class='menu-nav'>
+      <a href="index.html">
+        <img src="images/logo_1.png" alt="Logo" class='logo-image'>
+      </a>
+      <ul class='menu-list'>
+        <li class="menu-item"><a href="about.html" class='nav-link'>About</a></li>
+        <li class="menu-item"><a href="index.html" class='nav-link'>Program</a></li>
```

Comment:
- [ ] it would be more helpful for the users if the text matches the destination web page i.e if the link text is **Home** instead of **program**